pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -1,13 +1,13 @@
 {
   "dependencies": {
     "Microsoft.NETCore.Portable.Compatibility": "1.0.2",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.3",
-    "Microsoft.NETCore.Targets": "1.0.2",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.5-servicing-24601-01",
+    "Microsoft.NETCore.Targets": "1.0.1",
     "Microsoft.Win32.Primitives": "4.0.1",
     "System.ComponentModel.EventBasedAsync": "4.0.11",
     "System.Data.Common": "4.1.0",
     "System.Diagnostics.Contracts": "4.0.1",
-    "System.Diagnostics.StackTrace": "4.0.2",
+    "System.Diagnostics.StackTrace": "4.0.1",
```

Comment:
Downgraded `System.Diagnostics.StackTrace` and `Microsoft.NETCore.Targets` because of contents of https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/1.0.0/Latest_Packages.txt (had a higher version before this auto-PR because of manual upgrade in https://github.com/dotnet/core-setup/pull/338.)